packages/assert/scripts/postpublish.js

```diff
@@ -3,12 +3,13 @@ const packageJSON = require('../package.json');
 
```

Comment:
should we add files with the 'js' extension to the top level 'yarn prettier' command ?